code/ATMOSPHERICS/hvac/geyser.dm

```diff
@@ -0,0 +1,46 @@
+#define GEYSER_TEMP 40
+#define GEYSER_POWER 40000
+#define GEYSER_HEAT_RANGE 3
+#define GEYSER_WARM_RATE 25
+
+/obj/structure/geyser
+	name = "thermal geyser"
+	desc = "A geyser is a type of spring that is caused by the upward force of heat attemping to escape outward from a mineral body."
+	density = 1
+	anchored = 1
+	icon = 'icons/misc/beach.dmi'
+	icon_state = "geyser"
+
+/obj/structure/geyser/process()
```

Comment:
Why not take a leaf out of IRL geyser behaviour?

Have a period of activity, and a period of inactivity. Then, when it's active, you have the tell be it producing smoke.

Could go full ONI and have it expel certain gases

Reply:
I want it to be a consistently warm area so jackets are not needed.

Reply:
Sounds like a fancy fire

Reply:
Perhaps thermal vent makes more sense

Reply:
Right now it just seems incredibly casual if it's just going to heat people up magically

Reply:
Heating the atmosphere won't work since most of snowmap is unsimmed, but as you can see I have a provision in case it is in a simmed area

Reply:
- It doesn't take your clothing into account, it just warms you up while you're in the vicinity
 - It increases your temperature so quickly you'd go into shock IRL

This is casual as hell. Develop it for goodness sake

Reply:
It's supposed to take clothing into account, but the check needs to be inverted.
The heating is based off the rate of coffee.

Reply:
Right now you've got an if statement to check if they have protection. This doesn't lessen the effect of the heating at all